daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/StablePackage.scala

```diff
@@ -29,6 +29,17 @@ private[daml] sealed class StablePackage(
 private[daml] object StablePackage {
 
   object DA {
+    object Action {
+      object State {
+        object Type
+            extends StablePackage(
+              "DA.Action.State.Type",
+              "10e0333b52bba1ff147fc408a6b7d68465b157635ee230493bd6029b750dcb05",
+              "daml-stdlib",
+              v1_14,
```

Comment:
Why 14 ?

Reply:
because it's the earliest supported output LF version, i.e. there's no way for a package targeting e.g. LF 1.13 to use it